tests/parsers/apache_access.py

```diff
@@ -100,5 +100,37 @@ def testParse(self):
     self.assertEqual(warning.parser_chain, 'apache_access')
```

Comment:
add some check for the warning string

Reply:
I don't understand what you mean, the message string is checked in the line above. Please clarify.

Reply:
I'm referring to the warning `self.assertEqual(storage_writer.number_of_warnings, 1)`

I'm asking to add a check of the warning (message) string so that we know what it is expected to fail on.

Reply:
Line 97 is "self.assertEqual(warning.message, (...."  - this already checked.

Reply:
ack, I see, thix is the github UI issue again